src/Login.tsx

```diff
@@ -30,7 +30,10 @@ const Login: React.FC = () => {
   const [errors, setErrors] = useState('')
   const history = useHistory()
 
-  const attemptLogin = (setToken?: (token: string) => void) => () => {
+  const { setToken } = useContext(AuthContext)
+
+  const attemptLogin = (ev: React.FormEvent): void => {
+    ev.preventDefault()
```

Comment:
@DLavin23 This is the real fix for the issue you saw.  Because we introduced `<Box as="form">..." below, clicking on the `Button` was making the browser go about its normal form submission thing, so the page would refresh to do a form submit and the `attemptLogin` code wouldn't run at all.  Thus `ev.preventDevault()` to ask the browser to let us handle our own shit.

Reply:
ahhh I didn't even think about that. Was just focusing on slapping together some semantic and accessible HTML.